spec/offchain.tex

```diff
@@ -163,9 +170,9 @@ \subsubsection{Initializing the head}
 transaction, the parties compute $\Uinit \gets \bigcup_{j=1}^{n} C_j$ using previously
 observed $C_j$ and initialize $\hatmL = \Uinit$. The seen transaction set is
 initialized empty $\hatmT = \emptyset$, seen head state version $\hatv = 0$, as
-well as snapshot number $\hats = 0$. No commit transaction is pending
-$\tx_{\omega} = \bot$ and the initial snapshot object is defined accordingly
-$\bar{\mc S} \gets \blue{\Sno(0, 0, \Uinit, \emptyset, \bot)}$.
+well as snapshot number $\hats = 0$. No \red{increment/}decrement UTxO is pending
+($U_\alpha = \bot \land  U_\omega = \bot$) and the initial snapshot object is defined accordingly
```

Comment:
Are we keeping now UTxOs or transactions in our state?

Here we mention $U_\alpha$, in the figure it is $tx_{\omega}$ and the "Variables" section explains both.

Must: keep only transaction **or** UTxO